src/printing.py

```diff
@@ -11,7 +11,12 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print ('x is %(x)d ,y is %(y)3.2f ,z is %(z)s' %
```

Comment:
x, y, & z are already declared for you up top. you can simply do
print("x is %d, y is %.2f, z is \"%s\"" % (x, y, z))